build.sbt

```diff
@@ -320,29 +323,73 @@ lazy val cli = (project in file("examples") / "cli")
     Test / javaOptions += s"-Duser.dir=${(ThisBuild / baseDirectory).value.getAbsolutePath}",
   )
 
+lazy val serverAndClientShaded = (project in file("server-and-client-shaded"))
```

Comment:
can you leave comment on what this project is used for.

Reply:
also mention that this combines test classes as well.

Reply:
Sure, had added to the PR description, will add here as well.

Reply:
Added.